rel-eng/push-packages-to-obs.sh

```diff
@@ -272,6 +272,7 @@ while read PKG_NAME; do
           # SUSE Manager settings
           VERSION=$(sed 's/^\([0-9]\+\.[0-9]\+\).*$/\1/' ${BASE_DIR}/packages/uyuni-base)
           sed "s/^ARG INIT_BASE=.*$/ARG INIT_BASE=bci\/bci-init:15.4/" -i $SRPM_PKG_DIR/Dockerfile
+          sed "s/^ARG PRODUCT_PATTERN_PREFIX=.*$/ARG PRODUCT_PATTERN_PREFIX=patterns-suma/" -i $SRPM_PKG_DIR/Dockerfile
```

Comment:
here  we have INIT_BASE, but in the Dockerfile you added INIT_IMAGE for  server-image.  are  you sure?